README.md

```diff
@@ -69,7 +69,28 @@ Some of the inputs are tags. All infrastructure resources need to be tagged acco
 
 The `user_policy` input is useful when migrating data from existing bucket(s). For commands like `s3 ls` or `s3 sync` to work across accounts, a policy granting access must be set in 2 places: the *source bucket* and the *destination user*
 
-Example for the source bucket:
+
+### Source policy
+
+The source bucket must permit the destination s3 IAM user to "read" from its bucket explcitly.
+
+Example to retrieve destination IAM user for use in source bucket policy
+
+```bash
+# retrieve destination s3 user ARN
+$ unset AWS_PROFILE ; read K a n S <<<$(kubectl -n my-namespace get secret my-s3-secrets -o json | jq -r '.data[] | @base64d') ; export AWS_ACCESS_KEY_ID=$K ; export AWS_SECRET_ACCESS_KEY=$S ; aws sts get-caller-identity
```

Comment:
the 'read K a n S <<<...' part doesn't work for me. Are you sure that's correct?

Reply:
^ it's definitely correct, and is a bash-ism, maybe won't work in other shells